corehq/apps/reports/standard/monitoring.py

```diff
@@ -965,8 +965,12 @@ def get_row(self, user=None):
 
     def get_raw_user_link(self, user):
         from corehq.apps.reports.standard.inspect import SubmitHistory
+        sub_time_param = {
+            CompletionOrSubmissionTimeFilter.slug:
+            CompletionOrSubmissionTimeFilter.get_value(self.request, self.domain) or ''
+        }
```

Comment:
Would it be better to not supply the extra parameter if it has no value? I think it would make the link shorter, which seems like a good thing.
```suggestion
        value = CompletionOrSubmissionTimeFilter.get_value(self.request, self.domain)
        sub_time_param = {CompletionOrSubmissionTimeFilter.slug: value} if value else None
```